fcrepo-configs/src/main/java/org/fcrepo/config/OcflPropsConfig.java

```diff
@@ -92,12 +92,31 @@
     @Value("${fcrepo.resource-header-cache.expire-after-seconds:600}")
     private long resourceHeadersCacheExpireAfterSeconds;
 
+    @Value("${fcrepo.ocfl.reindex.threads:-1}")
```

Comment:
We will need to either document these configuration elements in the [wiki](https://wiki.lyrasis.org/display/FEDORA6x/Configuration+Options+Inventory) now, or as a follow-on ticket.

Reply:
Updated
https://wiki.lyrasis.org/display/FEDORA6x/Configuration+Options+Inventory#ConfigurationOptionsInventory-RebuildfromOCFLpersistentstorage